devices/Ssd13xx/RotationOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Ssd13xx
+{
+    /// <summary>
+    /// Options for rotation of the screen.
+    /// </summary>
+    public enum RotationOptions
```

Comment:
Just to be sure why you decided to go for rotation rather than the display orientation: is it because the original display can be different?
And why not remapping with the display orientation I proposed?
I4m asking as we're currently thinking of extracting some core display feature to work with both the pure managed drivers we have (like this one) and the native ones.

Reply:
The displays have a default orientation that's sometimes confusing.  Most are oriented to landscape by default but some (like the Adafruit FeatherWing) is oriented in portrait by default.  I could map it to the ones you proposed but I wasn't 100% confident that a screen that's set up as 64x128 is always portrait oriented by default and one that is 128x64 is always oriented landscape by default.  It would make sense, but I wasn't 100% sure.  That's why I opted for the enum that's in reference to whatever the default orientation happens to be.  That way should there be some weird scenario, it would still make sense.  

With all of that being said, I can go by that assumption.  It's an easy change if that's better for other needs.

Reply:
OK.  It took some figuring out, but it now supports absolute orientations.  To do this, I make the assumption that if the width < height then the default orientation is portrait.  I figure that's a safe assumption.  From there I figure out how to rotate by counting the 90 degree rotations necessary from the assumed default orientation.

Reply:
Also found a bug in the text centering when rotating from a default portrait orientation to a landscape orientation.  Fixed.